subsys/partition_manager/pm.yml.tfm

```diff
@@ -0,0 +1,36 @@
+#include <autoconf.h>
+
+tfm_ram:
+  placement: {after: [start]}
+  size: CONFIG_PM_PARTITION_SIZE_TFM_RAM
+  region: sram_primary
```

Comment:
Don't you need to split the TFM ram from the non secure ram?

Reply:
I assume that the limits are hard coded by TFM anyway, and not handled like it is in SPM.

That difference might deserve a comment if people are wondering why SPM use a separate region while TFM does not.

Reply:
Sure, I'm thinking of removing the prompt from the Kconfigs and explaining in the help text. Or do you think a comment here is necessary instead/also?

Reply:
Modified the Kconfigs, take a look.